data/pages-content/cfp.md

```diff
@@ -0,0 +1,164 @@
+---
+title: CFP
+subtitle: EuroPython Call for Proposals
+---
+# Call for Proposals
+
+<div style={{textAlign: "center",  marginBottom: 10}}>
+**EuroPython 2024 Call for Proposals (CFP) is Open.**
+
+It will be running between
+**Wednesday, 14th February 2024** and **Wednesday, 6th March 2024**
+[Anywhere on Earth](https://en.wikipedia.org/wiki/Anywhere_on_Earth)
+</div>
+
+---
+
+EuroPython reflects the colourful and diverse backgrounds, cultures and
+interests of our community, so you (yes, you!) should go for it: **propose
+something and represent!**
+
+No matter your level of Python or public speaking experience, EuroPython's job
+is to help you bring yourself to our community so we all flourish and benefit
+from each other's experience and contribution.
+
+If you’re thinking, “but they don’t mean me”, then we especially mean **YOU**.
+
+* If you’re from a background that isn't usually well-represented in most
+  Python groups, get involved - we want to help you make a difference.
+
+* If you’re from a background which is well-represented in most Python groups,
+  get involved - we want your help making a difference.
+
+* If you’re worried about not being technical enough, get involved - your fresh
+  perspective will be invaluable.
+
+* If you think you’re an imposter, join us - many of the EuroPython organisers
+  feel this way.
+
+* This is a volunteer led community, so join us - you will be welcomed with
+  friendship, support and compassion.
+
+You are welcome to share your questions and ideas with our programme team at
+[programme@europython.eu](mailto:programme@europython.eu).
+
+## The Conference Format ##
+
+The conference will feature around 16 Tutorials/Workshops and 120 talk sessions:
+
+* **Tutorials/Workshops (July 8th and 9th)**: hands-on **180-mins sessions** with
```

Comment:
```suggestion
* **Tutorials/workshops (July 8th and 9th)**: hands-on **180 minute sessions** with
```

(I'd say to either use “min” on both bullet points or “minute” on both bullet points, but not one of each.)